lib/src/native/realm_core.dart

```diff
@@ -457,6 +497,12 @@ class RealmListHandle extends Handle<realm_list> {
   }
 }
 
+class RealmQueryHandle extends Handle<realm_query> {
+  RealmQueryHandle._(Pointer<realm_query> pointer) : super(pointer) {
+    _realmLib.realm_attach_finalizer(this, _pointer.cast(), 88); // TODO!
```

Comment:
What is TODO?
88?

Reply:
It is a hint to the GC. 

Handles are are dart objects representing object in the native heap. These are not released until the corresponding dart handle is released. Since handles are tiny objects, the GC does not have a lot of pressure to release them, but they may keep large object alive in the native heap. 

88 is an estimate of how many bytes this handles keeps alive in the native heap. The GC will consider this when cleaning the various generations.

The TODO means I just slapped something in there. We should tune it - not necessarily to the correct number, but one that balances how aggressive the GC will free the handle. 

Note, that the ffi helper sizeOf cannot be used here, since `Pointer<realm_query>` is opaque (basically a void pointer)

Reply:
It's a really cool feature that other GC's don't have. My point was merely to make a note about what is TODO. E.g. "Reevaluate the 88 hint"